### ISSUE TITLE
Close hourly forecast -button can't be pressed with larger fonts

### ISSUE THREAD

**Issue by AnttiRumpunen** [fmidev/weather-app#1270]
x -button is a bit too high...

Also X-button can cover > -button so user cannot navigate to next day.

<img width="1125" height="2436" alt="Image" src="https://github.com/user-attachments/assets/13da5ed1-3665-47a4-81cb-a81238ac7082" />

**Comment by geosaaga** [comment]
PR: https://github.com/fmidev/weather-app/pull/1302
